cypress/integration/plugins/index-management-dashboards-plugin/rollups_spec.js

```diff
@@ -242,28 +242,37 @@ describe("Rollups", () => {
       // Confirm we have our initial rollup
       cy.contains(ROLLUP_ID);
 
+      // Intercept different rollups requests endpoints to wait before clicking disable and enable buttons
+      cy.intercept(`/api/ism/rollups/${ROLLUP_ID}`).as("getRollup");
+      cy.intercept(`/api/ism/rollups/${ROLLUP_ID}/_stop`).as("stopRollup");
+
       // Click into rollup job details page
       cy.get(`[data-test-subj="rollupLink_${ROLLUP_ID}"]`).click({
         force: true,
       });
 
       cy.contains(`${ROLLUP_ID}`);
 
-      // Disable button is enabled
-      cy.get(`[data-test-subj="disableButton"]`).should("not.be.disabled");
+      cy.wait("@getRollup").wait(2000);
```

Comment:
I suppose getRollup will be triggered after line 250 click the rollup link, wondering is there a chance to already miss the getRollup call at this line.

Is there a way to chain this with line 250 like 
```
cy.get(`[data-test-subj="rollupLink_${ROLLUP_ID}"]`).click({
        force: true,
      }).wait("@getRollup");
```

Reply:
It should not be missed as [cypress waits for the request/response cycle to complete.](https://docs.cypress.io/api/commands/intercept#Waiting-on-a-request).